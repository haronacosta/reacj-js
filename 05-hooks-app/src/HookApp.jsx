import React from 'react';

const HookApp = () => {
  return <div className='h1'>Hello</div>;
};

export default HookApp;
