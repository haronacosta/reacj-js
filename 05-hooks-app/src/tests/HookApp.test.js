import React from 'react';
import { shallow } from 'enzyme';
import HookApp from '../HookApp';

describe('Pruebas al componente <HookApp/>', () => {
  test('Se debe de mostrar correctamete el componente', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
