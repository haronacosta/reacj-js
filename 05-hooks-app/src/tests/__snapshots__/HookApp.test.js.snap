// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas al componente <HookApp/> Se debe de mostrar correctamete el componente 1`] = `
<div
  className="h1"
>
  Hello
</div>
`;
